pkg/aws/policies.go

```diff
@@ -238,6 +238,24 @@ func (c *awsClient) EnsureRole(name string, policy string, permissionsBoundary s
 	return roleArn, nil
 }
 
+func (c *awsClient) ValidateRoleNameAvailable(name string) (err error) {
+	_, err = c.iamClient.GetRole(&iam.GetRoleInput{
+		RoleName: aws.String(name),
+	})
+	if err == nil {
+		// If we found an existing role with this name we want to error
+		return fmt.Errorf("A role named '%s' already exists", name)
```

Comment:
should we give some action for the user to take here? Feels like the error is kinda confusing, would it be better to return an info message with some options to either create cluster with different role prefix, or delete existing role prefix if its not in use :thinking:

Reply:
Something like 'Please delete the existing role, or use a different prefix'?

Reply:
yeah i think its a common pattern in the cli, to provide some [next steps](https://github.com/openshift/rosa/blob/master/cmd/create/cluster/cmd.go#L1701-L1721) especially if it can be solved with the CLI

Reply:
fixed